readme.md

```diff
@@ -1,13 +1,40 @@
-# React Widget
+# File Uploader for React
+
+<a href="https://uploadcare.com/?utm_source=github&utm_campaign=uploadcare-tinymce">
+  <img align="right" width="64" height="64"
+    src="https://ucarecdn.com/2f4864b7-ed0e-4411-965b-8148623aa680/uploadcare-logo-mark.svg"
+    alt="">
+</a>
+
+This is a React component made to easily integrate
+[Uploadcare Widget][uc-feature-widget] into your React app; convenient props
+management and lazy loading are bundled.
+
+The component allows users to upload media from their devices, social media,
+cloud storage, and more. All that without any backend code that's usually
+required to handle uploads.
+
+**Note: This library comes untranspiled**. It means, if you want to support
+IE11, make sure you transpile `node_modules`. → [Read More][es6-debate] ←
```

Comment:
Is it really **THAT IMPORTANT**, couldn't we just stick with `Note, blabla, [learn more](<source>)`?